OpenRA.Mods.Common/Traits/StoresPlayerResources.cs

```diff
@@ -0,0 +1,68 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Primitives;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Adds capacity to a player's harvested resource limit.")]
+	public class StoresPlayerResourcesInfo : TraitInfo
```

Comment:
Fine. Just makes me wonder. Is it a resource of the actor, player or map. 
Is it a single resource type with a (monetary) value. Or is each i.e. gem an instance of resource that you might like to store by itself.

Reply:
Those shouldn't depend on both the trait that adds resources to storage (i.e. Harvester) and the trait that spends it (i.e. Production)